packages/picasso.js/src/web/components/brush-range/brush-area-dir.js

```diff
@@ -172,16 +172,10 @@ const brushAreaDirectionalComponent = {
     areaClear(e) { this.clear(e); }
   },
   created() {
-    this.rect = {
-      x: 0, y: 0, width: 0, height: 0
-    };
     this.state = {
       key: this.settings.key || 'brush-area-dir'
     };
   },
-  beforeRender(opts) {
-    this.rect = opts.size.computedInner;
```

Comment:
Not using computedInner will likely break the component when the dock layout is scaled